src/components/MapView/Layers/BoundaryDropdown.tsx

```diff
@@ -299,8 +333,91 @@ function BoundaryDropdown({
       setSelectedBoundaries={newSelectedBoundaries => {
         dispatch(setSelectedBoundariesRedux(newSelectedBoundaries));
       }}
+      labelMessage={labelMessage}
     />
   );
 }
 
+export const SelectMenu = withStyles(() => {
```

Comment:
Is there a better name that summarizes what exactly you're changing here? Or a comment would help since this isn't too straight forward from an initial read.

Reply:
Included in a087321458770c698a09ed29d8823566cbaf9e01 and also in previous comment